client/src/app/site/users/components/group-list/group-list.component.html

```diff
@@ -85,7 +85,7 @@ <h1 mat-dialog-title>
         <button type="button" mat-button (click)="cancelEditing()">
             <span translate>Cancel</span>
         </button>
-        <button *ngIf="selectedGroup" type="button" mat-icon-button color="warn" (click)="deleteSelectedGroup()">
+        <button *ngIf="selectedGroup && canDeleteGroup(selectedGroup)" type="button" mat-icon-button color="warn" (click)="deleteSelectedGroup()">
```

Comment:
The mat icon button seems to be pretty misplaced here
```suggestion
        <button *ngIf="selectedGroup" [disabled]="!canDeleteGroup(selectedGroup)" type="button" mat-button color="warn" (click)="deleteSelectedGroup()">
         <span translate>Delete</span>
```